routes/web.php

```diff
@@ -29,3 +35,23 @@
 Route::post("/user/change-password", [ChangePasswordController::class, "store"])->name("user.changePassword.store");
 
 Route::post("/logout", [LogoutController::class, "logout"])->name("auth.logout");
+
+Route::middleware("auth")->group(function (): void {
+    Route::get("/auction", [AuctionController::class, "create"])->name("auction.create");
+    Route::post("/auction", [AuctionController::class, "store"])->name("auction.store");
```

Comment:
W dokumentacji laravela masz ładnie opisane jak powinny wyglądać routy
https://laravel.com/docs/12.x/controllers#actions-handled-by-resource-controllers

```suggestion
    Route::get("/auctions/create", [AuctionController::class, "create"])->name("auction.create");
    Route::post("/auctions", [AuctionController::class, "store"])->name("auction.store");
```